docs/how-to/enable-doppelganger-detection.md

```diff
@@ -41,9 +41,9 @@ This means that the validators being checked are inactive for at most two epochs
 
 Keeping the validators inactive might cause:
 
-- Missed attestations.
-- Missed sync committee contributions.
-- Missed block proposals.
+- Missed attestations
+- Missed sync committee contributions
+- Missed block proposals
```

Comment:
```suggestion
- Missed attestations.
- Missed sync committee contributions.
- Missed block proposals.
```

Please review the list guidelines from the Microsoft Style Guide: https://learn.microsoft.com/en-us/style-guide/scannable-content/lists. If items complete a sentence, they should end with a period.